ganga/GangaCore/GPIDev/Base/Objects.py

```diff
@@ -25,6 +25,8 @@
 
 from GangaCore.Utility.Plugin import allPlugins
 
+from GangaCore.Utility.Monitoring import cpu_profiler, mem_profiler
```

Comment:
Where is the cpu_profiler package from. I did not find it with pip.

Reply:
@egede  cProfile comes pre-installed with python. [docs](https://docs.python.org/2/library/profile.html)